pymc/logprob/transforms.py

```diff
@@ -563,7 +621,12 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
 
     transform_dict = {
         Exp: ExpTransform(),
+        Exp2: ExpTransform(base=2),
+        Expm1: ExpTransform(m=True),
         Log: LogTransform(),
+        Log2: LogTransform(base=2),
```

Comment:
I was thinking of converting stuff like `log2(x) -> log(x) / log(2)` . This way we don't need to implement the new transforms

Reply:
I think it can also be done for exponents?

Reply:
I see! Just to check my understanding then, I can add to this function
```python 
@node_rewriter([log1p, softplus, log1mexp])
def measurable_special_log_to_log(fgraph, node):
    """Convert log1p, log1mexp, softplus of `MeasurableVariable`s to log form."""
    [inp] = node.inputs

    if isinstance(node.op.scalar_op, Log1p):
        return [pt.log(1 + inp)]
    if isinstance(node.op.scalar_op, Softplus):
        return [pt.log(1 + pt.exp(inp))]
    if isinstance(node.op.scalar_op, Log1mexp):
        return [pt.log(1 - pt.exp(pt.neg(inp)))]
    if isinstance(node.op.scalar_op, Log2):
        return pt.log(inp)/pt.log(2)
```

Reply:
Yes 👍